### DISCUSSION TITLE
Embed Tabs inside Rows and Columns

### DISCUSSION THREAD

**Discussion by durchgedreht** [apache/superset#14279] [Ideas]
Then dealing with multiple tabs I sometimes need to have the same filter for all charts. Thus it would be great for the user to have the FilterBox sticky beside the tabs (you can just put it inside a single tab, above or below).

The solution from a user persepctive would be either:

1. Embed the complete filterbox in a single table element
2. Make a single Row/Column available (left or right) from the tab element 

Example of filter inside tab (selecting B will move filter out of viewable area):
<img width="1163" alt="Screen Shot 2020-10-29 at 18 18 08" src="https://user-images.githubusercontent.com/4490840/97609127-826fd500-1a13-11eb-8e6d-150eba4bee1e.png">

Goal: Move tabs in empty box (or other way round - optionally leave a space left from the tabs):
<img width="1172" alt="Screen Shot 2020-10-29 at 18 20 25" src="https://user-images.githubusercontent.com/4490840/97609563-0c1fa280-1a14-11eb-963a-50bcacee19fb.png">

[a comment by an automated account is not shown]

**Comment by junlincc** [comment]
Right now, filters are at the dashboard level despite the number of layers of tab in the dashboard. Meaning changing filter values in the filter box and chart filtering action could happen on different pages, which creates confusion for the dashboard consumer. The UX certainly has room for improvement. 
We will be implementing dashboard native filter soon and may not further invest in the current filter box solution by adding new features to it. Though all filters will still be placed at the global/dashboard level, hopefully the confusion will be eliminated by the new UI.  
Thanks for suggesting!
